### PR TITLE
Product list page

### PR DESCRIPTION
**Pull request by Ramandeep9898**

### CODE REVIEW

File: src/components/Aside.component/Aside.jsx [hunk 1]
```diff
@@ -0,0 +1,231 @@
+import react from "react";
+import { useReducer } from "react";
+import "./aside.css";
+import { useProduct } from "../ProductCard.Component/productContext";
+
+const Aside = () => {
```
Comment (devsurajrai) [inline src/components/Aside.component/Aside.jsx:6]: Component naming could be more descriptive

File: src/components/Aside.component/Aside.jsx [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+  const { state, dispatch } = useProduct();
+  const { sortBy, discount, categories } = state;
+  const { bestseller, fiction, nonfiction, horror } = categories;
+
+  return (
+    <aside className="product-page-aside">
```
Comment (devsurajrai) [inline src/components/Aside.component/Aside.jsx:12]: good to see that you are using semantic html.

File: src/components/ProductCard.Component/ProductCard.jsx [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+                </button>
+                <img src={img} alt="" className="avatar sq-avatar card-img" />
+                <div className="card-typo">
+                  <div className="card capitalize fW-400 color text-left">
+                    {title}
+                  </div>
+                  <div className="card-subtitle capitalize fW-400 color text-left mgT-4">
+                    -by {author}
+                  </div>
+                  <div className="card capitalize fW-500 color text-left mgT-4">
```
Comment (devsurajrai) [inline src/components/ProductCard.Component/ProductCard.jsx:61]: It would be nice if you just abstract them away .

**Review by devsurajrai — COMMENTED** [review]
Everything seems good. Awesome Work. Just try to apply the changes suggested.

**Review by yashodaAgarwal — COMMENTED** [review]
Code looks clean. class names are good.
Good to merge, just added some suggestions.

**Pull request merged**